tsconfig.base.json

```
@@ -2,9 +2,9 @@
 	"compilerOptions": {
 		"composite": true,
 		"incremental": true,
-		"target": "ES2020",
+		"target": "ES2022",
 		"module": "commonjs",
```

```suggestion
		"module": "nodenext",
```

I ran into a few TypeScript errors related to the stylelint type declarations:

```
src/server/services/documents/document-fixes.service.ts:1:28 - error TS2307: Cannot find module 'stylelint' or its corresponding type declarations.
  There are types at '/Users/jeddy3/Projects/vscode-stylelint/node_modules/stylelint/types/stylelint/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.

1 import type stylelint from 'stylelint';
                             ~~~~~~~~~~~
```

We changed Stylelint's Typescript config in https://github.com/stylelint/stylelint/pull/8875 and how we export the types in https://github.com/stylelint/stylelint/pull/8859.

Changing the `module` property to `nodenext` resolved the issue, but is this something we need to consider in Stylelint?